src/idl_gen_cpp.cpp

```diff
@@ -719,6 +723,71 @@ class CppGenerator : public BaseGenerator {
       }
     }
     code += "  return builder_.Finish();\n}\n\n";
+
+    //Generate a CreateX function with vector types as parameters
+    if (gen_vector_pars) {
```

Comment:
Please run `generate_code.sh|.bat` in `tests` to include the changed generated code in this commit, so we can see what this code changes.